packages/vitest/src/api/setup.ts

```diff
@@ -88,13 +93,11 @@ export function setup(vitestOrWorkspace: Vitest | WorkspaceProject, server?: Vit
           return fs.writeFile(id, content, 'utf-8')
         },
         async saveSnapshotFile(id, content) {
-          if (!ctx.snapshot.resolvedPaths.has(id))
-            throw new Error(`Snapshot file "${id}" does not exist.`)
           await fs.mkdir(dirname(id), { recursive: true })
```

Comment:
This allows saving any file on the file system (including overwriting system configs)

Reply:
Doesn't sound great! Will revert.

Reply:
Yes, I thought you would do that

Reply:
Done!